File: conexiones/conexionMongo.java
import com.mongodb.MongoClient;
import com.mongodb.MongoClientURI;

public class conexionMongo {
    public static void main(String[] args) {
        try {
            MongoClientURI uri = new MongoClientURI("mongodb://localhost:27017/");
            MongoClient mongoClient = new MongoClient(uri);
            System.out.println("Conexion exitosa a MongoDB");
            mongoClient.close();
        } catch(Exception e) {
            System.out.println("Error: " + e.getMessage());
        }
    }
}

File: conexiones/conexionMongo.rb
require 'mongo'
include Mongo

begin
  # Conecta al cliente de MongoDB y selecciona la base de datos "ecommerce2996948"
  client = Mongo::Client.new("mongodb://localhost:27017/ecommerce2996948")
  puts "Conexión exitosa a MongoDB"
rescue => e
  puts "Error de conexión: #{e.message}"
end
